util/microservice-sdk/js/examples/ConfigManager/README.md

```diff
@@ -0,0 +1,18 @@
+# Config Manager
```

Comment:
Maybe in this file you can explican how to run docker-compose and see everything happen.